### PR TITLE
milestone 1: process of creating your portfolio website

### PR DESCRIPTION
**Pull request by banlon-jones**
# Creating my portfolio website
in the process of creating my portfolio website i
- added HTML and CSS linter. 
- added .gitignore file.
- added the index.html file.
- added styles.css stylesheet.
- added images and icons in the images folder.
- develop the portfolio header(navbar).
- develop the portfolio headline section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <title> home </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./css/styles.css"/>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <div class="brand">
+                    <h1> My Logo </h1>
+                </div>
```
Comment (keneogu) [inline index.html]: The logo should also be a link... you can make it a link by using the `<a>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <section class="intro">
+            <div class="title">
+                <h1> I’m Mariane Glad to see you! </h1>               
+            </div>
+            <div>
+                <article>
+                    I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </article>
```
Comment (keneogu) [inline index.html]: This is a paragraph. Make use of the `<p>` tag instead of the article semantic tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                <div>
+                    <img src="./images/medium.svg" alt="medium logo"/>
+                </div>
+                <div>
+                    <img src="./images/github.svg" alt="github logo"/>
+                </div>
+                <div>
+                    <img src="./images/Vector.svg" alt="vector logo"/>
+                </div>
+            </section>
```
Comment (keneogu) [inline index.html]: Use the `<ul>` tag to group the list of icons and `<li>` per each icon not `<div>` tag. Each icon should be a link: you can make it a link by using the `<a>` tag.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @banlon-jones , 

Good job so far!
The linters works fine.
Pull request has a descriptive title.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Move the logo out of the nav tag

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    <a href="#">
+                        <img src="./images/github.svg" alt="github logo"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Vector.svg" alt="vector logo"/>
+                    </a>
+                </li>
+            </ul>
```
Comment (rotimiazeez) [inline index.html:60]: - [ ] Kindly make sure the icons are well aligned as seen in the original template😉✌
- Figma design
screenshot (3)
- Your design
local design

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @banlon-jones 👋, 

### Great work on making the changes requested by a previous reviewer 👏 
1[](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @banlon-jones, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**